test/common/filter/config_discovery_impl_test.cc

```diff
@@ -610,14 +613,16 @@ TYPED_TEST(FilterConfigDiscoveryImplTestParameter, TerminalFilterInvalid) {
 
   if (config_discovery_test.getFilterType() == "listener" ||
       config_discovery_test.getFilterType() == "upstream_network") {
-    EXPECT_NO_THROW(config_discovery_test.callbacks_->onConfigUpdate(decoded_resources.refvec_,
-                                                                     response.version_info()));
+    ASSERT_TRUE(config_discovery_test.callbacks_
+                    ->onConfigUpdate(decoded_resources.refvec_, response.version_info())
+                    .ok());
     return;
   }
 
   EXPECT_THROW_WITH_MESSAGE(
-      config_discovery_test.callbacks_->onConfigUpdate(decoded_resources.refvec_,
-                                                       response.version_info()),
+      EXPECT_TRUE(config_discovery_test.callbacks_
```

Comment:
A bit confusing (expect throw, while expecting a successful return value), but I guess this is just to satisfy the linter.

Reply:
yeah and they're mostly all changing in the follow-up PR.